### PR TITLE
Tick style

### PR DESCRIPTION
**Pull request by maasencioh**
- add tickStyle to the axis, allowing to add styles (uses kebab-case)
- add the source to the stories
- fix the default color palette error

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -36,6 +37,7 @@ interface AxisProps {
   label?: string;
   showGridLines?: boolean;
   labelStyle?: CSSProperties;
+  tickStyle?: PropertiesHyphen;
```
Comment (targos) [inline src/types.ts:40]: I don't like this because it is different from the other style properties. I'm sure we can avoid it.

File: package.json [hunk 2]
```diff
@@ -69,6 +69,8 @@
     "typescript": "^4.0.3"
   },
   "dependencies": {
+    "@storybook/addon-storysource": "^6.1.11",
```
Comment (targos) [inline package.json:72]: devDependencies

**Review by targos — APPROVED** [review]

File: src/utils.ts [hunk 3]
```diff
@@ -45,3 +45,7 @@ export function splitChildren(children: React.ReactNode): PlotChildren {
   }
   return { invalidChild, lineSeries, axis, heading, legend };
 }
+
+export function camelToKebab(string: string) {
+  return string.replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2').toLowerCase();
```
Comment (targos) [inline src/utils.ts:50]:
```suggestion
  // eslint-disable-next-line ...
  return string.replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2').toLowerCase();
```

Comment (maasencioh) [reply src/utils.ts:50]: I wasn't able to find how to do it with replaces, but here's the link if someone else wants it https://2ality.com/2017/05/regexp-named-capture-groups.html

**Pull request merged**